modules/backup/main.tf

```diff
@@ -113,3 +113,15 @@ resource "aws_backup_selection" "non_production" {
     }
   }
 }
+
+# SNS topic
+resource "aws_sns_topic" "backup_failure_topic" {
```

Comment:
the same comment as in the other PR, I wonder if it makes sense to define the sns topic where the backup live, rather than in the module (since it's not doing anything here anyway)

Reply:
Talked in stand up and went for, this could be useful for everyone